examples/hooks/components/Menu.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { useMenu, UseMenuProps } from 'react-instantsearch-hooks';
+
+import { cx } from '../cx';
+
+export type MenuProps = React.ComponentProps<'div'> & UseMenuProps;
+
+export function Menu(props: MenuProps) {
+  const {
+    canToggleShowMore,
+    isShowingMore,
+    items,
+    refine,
+    createURL,
+    toggleShowMore,
+  } = useMenu(props);
+
+  return (
+    <div className={cx('ais-Menu', props.className)}>
+      <ul className="ais-Menu-list">
+        {items.map((item) => (
+          <li
+            key={item.value}
+            className={cx(
+              'ais-Menu-item',
+              item.isRefined && 'ais-Menu-item--selected'
+            )}
+          >
+            <a
+              className="ais-Menu-link"
+              onClick={() => refine(item.value)}
```

Comment:
We don't need to prevent the default browser behavior? (and let special events do their behaviors)

Reply:
good call, I changed last second from inputs to a and forgot to prevent default

Reply:
done in https://github.com/algolia/react-instantsearch/pull/3197/commits/7d9978e67e755714b4e0d6bfdd9349a27f2ae22a#diff-6e1456b9e02009184c4ebc4b1fa6983a8b817d376884f200b2df6edc74fe87caR33-R36 now